packages/interactive_media_ads/example/lib/main.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_driver/driver_extension.dart';
+
+// ignore_for_file: public_member_api_docs
```

Comment:
I think for example apps intended for client reference (which app-facing package examples are) we should leave this enabled, and err on the side of over-commenting.